<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>currentlyListen</class>
 <widget class="QWidget" name="currentlyListen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Currently Playing Mix</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="grpCurrentplyPlaying">
     <property name="title">
      <string>Currently Playing Mix</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lblMixImg">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid #333;</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="lblMixName">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Nom du Mix</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblDescription">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>300</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="title">
               <string>Currently Playing ...</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="lblReleaseName">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblName">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblPerformer">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lblYear">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QPushButton" name="btnProfile">
                   <property name="text">
                    <string>Slug Utilisateur</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="btnPlayPause">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="mainWindow.qrc">
                 <normaloff>:/icons/Resources/Pause.png</normaloff>:/icons/Resources/Pause.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSkip">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="mainWindow.qrc">
                 <normaloff>:/icons/Resources/Gnome-Media-Skip-Forward-32.png</normaloff>:/icons/Resources/Gnome-Media-Skip-Forward-32.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="prgListenStatus">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>32</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="textVisible">
                <bool>true</bool>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="format">
                <string>%v s</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="volumeSlider">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>25</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>50</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
